tools/ros2_dependencies.repos

```diff
@@ -0,0 +1,9 @@
+repositories:
```

Comment:
can you move this to the root and call it ``moveit2.repos`` so it matches what will be there soon?

https://github.com/AcutronicRobotics/moveit2/blob/master/moveit2.repos